examples/README.md

```diff
@@ -0,0 +1,148 @@
+# JavaScript Client SDK
+
+_Read this in other languages_: [日本語](README.ja.md)
+
+<br>
+
+## Running the Example
+
+- Node version 16 or higher is required
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+
+```bash
+cd javascript
```

Comment:
```suggestion
cd examples
```

Reply:
I realized that this `README` is already in `examples` directory, so I removed `cd` command altogether.